CONTRIBUTING.md

```diff
@@ -26,9 +26,13 @@ Note: you will see warnings about `__dirname` and `require()` not being "a valid
 
 ```bash
 yarn build
-yarn --force
+yarn --force # only needed after very first build; afterward can be skipped
```

Comment:
If I recall correctly, we also have to call `yarn --force` when a new `plugin/*` folder has been created, as the binary symlink has to be created in that folder?

Reply:
We actually don’t! Plugins don’t need `node_modules` because they don’t run the `snowpack` CLI command.

But if the tests being added add the need for this, maybe we add a note in a followup?